tests/testMakeRawVisitInfo.py

```diff
@@ -267,6 +267,12 @@ def testPopMjdDate(self):
             missingDate = self.makeRawVisitInfo.popMjdDate(md, "BADKEY")
             self.assertEqual(missingDate, DateTime())
 
+    def testEraFromLstAndLongitude(self):
+        LST = 90*degrees
+        Longitude = 50*degrees
+        era = self.makeRawVisitInfo.eraFromLstAndLongitude(LST, Longitude)
+        self.assertEqual(era, LST-Longitude)
```

Comment:
I'm uneasy about testing exact equality here, since these are floats. Wouldn't assertAnglesAlmostEqual (or whatever it's called) be safer?